eagle-core/eagle-alert-parent/eagle-alert/alert-engine/src/main/java/org/apache/eagle/alert/engine/publisher/impl/AlertPublisherImpl.java

```diff
@@ -73,11 +76,11 @@ public void nextEvent(PublishPartition partition, AlertStreamEvent event) {
     private void notifyAlert(PublishPartition partition, AlertStreamEvent event) {
         // remove the column values for publish plugin match
         partition.getColumnValues().clear();
-        if (!publishPluginMapping.containsKey(partition)) {
+        if (!publishPluginMapping.containsKey(partition.getPublishId())) {
```

Comment:
In publisher, we are able to specify the accepted streamId list, here how can we handle dropping the alert which comes from the stream which is NOT defined accepted streamId list?